service/raft/raft_group0.cc

```diff
@@ -346,7 +358,9 @@ future<group0_info> persistent_discovery::run(
 }
 
 future<> raft_group0::abort() {
-    co_await uninit_rpc_verbs();
+    co_await smp::invoke_on_all([this]() {
+        return uninit_rpc_verbs();
```

Comment:
Maybe uninit_rpc_verbs() should be static, since raft_group0 is not a sharded object. Calling an instance method from different shards on it is unsafe in general. Currently it only accesses sharded _ms, but making it static would make this safer.

Reply:
made it static, along with `init_rpc_verbs`